roles/create-single-k8s-cluster-with-kubeadm/tasks/main.yml

```diff
@@ -27,9 +34,10 @@
       cat /etc/resolv.conf
 
       ## Initializing master
-      kubeadm init --pod-network-cidr=192.168.0.0/16
+      kubeadm init --pod-network-cidr=192.168.0.0/16 --kubernetes-version="{{ kubernetes_version }}"
```

Comment:
whether option `--kubernetes-version` support `latest` value?

Reply:
Yes， it supports， it will search the latest stable release, as the log test in allinone below: 
2019-03-18 02:22:57.760518 | ubuntu-xenial | + kubeadm init --pod-network-cidr=192.168.0.0/16 --kubernetes-version=latest
2019-03-18 02:22:58.877009 | ubuntu-xenial | I0318 02:22:58.876709    7552 version.go:237] remote version is much newer: v1.15.0-alpha.0; falling back to: stable-1.13
2019-03-18 02:22:59.851021 | ubuntu-xenial | [init] Using Kubernetes version: v1.13.4

Reply:
Thank you.